README.md

```diff
@@ -352,11 +352,11 @@ In order to track the pipelinerun progress run:
 tkn pipelinerun logs build-and-deploy-run-z2rz8 -f -n pipelines-tutorial
 ```
 
-Similarly, start a pipeline to build and deploy frontend application:
+Similarly, start a pipeline to build and deploy frontend application using `tkn`, but this time lets provide `VolumeClaimTemplate` instead of `PersistentVolumeClaim`:
 
 ```bash
 $ tkn pipeline start build-and-deploy \
-    -w name=shared-workspace,claimName=source-pvc \
+    -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
```

Comment:
Can we do the same changes here https://github.com/openshift/pipelines-tutorial/blob/master/02_pipelinerun/02_build_deploy_ui_pipelinerun.yaml#L17-L18 so that it will consistent while using `tkn` or `yaml` for `vote-ui`